src/cmdc_tools/datasets/official/MT/data.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
+import requests
+import textwrap
+
+from ..base import ArcGIS
+from ...base import DatasetBaseNoDate
+
+
+class Montana(ArcGIS, DatasetBaseNoDate):
+    def __init__(self, params=None):
+        self.ARCGIS_ID = "qnjIrwR8z5Izc0ij"
+
+        if params is None:
+            params = {
+                "f": "json",
+                "where": "1=1",
+                "outFields": "*",
+                "returnGeometry": "false",
+            }
+
+        super().__init__(params)
+
+    def _insert_query(self, df, table_name, temp_name, pk):
+        out = f"""
+        INSERT INTO data.{table_name} (vintage, dt, fips, variable_id, value)
+        SELECT tt.vintage, tt.dt, us.fips, mv.id as variable_id, tt.value
+        FROM {temp_name} tt
+        LEFT JOIN meta.us_fips us ON tt.county=us.name
+        LEFT JOIN meta.covid_variables mv ON tt.variable_name=mv.name
+        WHERE us.fips > 30000 AND us.fips < 31000
```

Comment:
Is there data for MT the state in this PR?

Reply:
No